packages/credentials/src/V1/KiltCredentialV1.ts

```diff
@@ -385,6 +448,43 @@ export async function validateSubject(
       [key.substring(vocab.length)]: value,
     }
   }, {})
-  // validates against CType (also validates CType schema itself)
-  CType.verifyClaimAgainstSchema(claims, cType)
+
+  // Connect to blockchain
+  const api = Kilt.ConfigService.get('api')
+
+  // Bizim eklediğimiz doğrulama mantığı
+  const isNested = cTypeTypeFinder(cType)
+  
+  if (!isNested) {
+    await CType.verifyClaimAgainstNestedSchemas(
+      cType,
+      [],
+      claims
+    )
+  } else {
+    const references = extractUniqueReferences(cType)
+    const referencedCTypes: ICType[] = []
+    
+    for (const ref of references) {
+      try {
+        const referencedCType = await CType.fetchFromChain(ref as any)
+        if (referencedCType.cType) {
+          referencedCTypes.push(referencedCType.cType)
+        }
```

Comment:
again, use the loadCTypes argument. If it can't be found, throw an error saying that the ctype contains a reference to another ctype which cannot be resolved.